### PR TITLE
[devops][api][SIMS #221] Institution Auth API Changes

### PR DESCRIPTION
**Pull request by fwpushan**
----------
1. Add new env variable for Institution keycloak client
2. Improve some global error messaging
3. Change local build tagging to static tagging with dev or env variable specific.
---
N.B: Part 1 / api part of the story.

### CODE REVIEW

File: sources/packages/web/src/services/AuthService.ts [hunk 1]
```diff
@@ -25,10 +25,10 @@ export default async function(
     });
     if (keycloak.authenticated) {
       switch (type) {
-        case "student":
+        case ClientIdType.STUDENT:
           store.dispatch("student/setStudentProfileData", keycloak);
           break;
-        case "institute":
+        case ClientIdType.INSTITUTE:
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/services/AuthService.ts:31]: We should rename INSTITUTE to INSTITUTION in all places as needed.

**Review by andrewsignori-aot — COMMENTED** [review]
Please rename INSTITUTE to INSTITUTION in all places as needed.

File: sources/packages/web/src/services/AuthService.ts (hunk 1 above)
Comment (fwpushan) [reply sources/packages/web/src/services/AuthService.ts:31]: I will do it in part 2

**Review by andrewsignori-aot — APPROVED** [review]
👍

**Review by 2011sandeepraj — APPROVED** [review]

**Pull request merged**